test/cql-pytest/run.py

```diff
@@ -236,7 +237,10 @@ def find_scylla():
         if not scyllas:
             print("Can't find a Scylla executable in {}.\nPlease build Scylla or set SCYLLA to the path of a Scylla executable.".format(source_path))
             exit(1)
-        scylla = max(scyllas, key=os.path.getmtime)
+        if len(scyllas) > 1:
+            print("Found several scylla executables and cannot chose one.\nPlease set SCYLLA to one of the following paths:\n{}".format('\n'.join(scyllas)))
+            exit(1)
```

Comment:
nit, 
```suggestion
            sys.exit(1)
```

> They are useful for the interactive interpreter shell and should not be used in programs.

see https://docs.python.org/3/library/constants.html?highlight=exit#exit

Reply:
Fair point. I made it similar to the pre-existing `exit(1)` above. Changing it deserves dedicated PR, I'll follow up if you don't mind

Reply:
sure.